File: PreEZ_segmentation_v1.5/addons/SpecificKeyListener.java
import java.awt.event.KeyAdapter;
import java.awt.event.KeyEvent;

public class SpecificKeyListener extends KeyAdapter {
    private boolean keyPressed = false;

    public SpecificKeyListener() {
        super();
    }

    @Override
    public void keyTyped(KeyEvent event) {
        if (event.getKeyChar() == 'x') {
            keyPressed = true;
        }
    }

    public boolean isKeyPressed() {
        return keyPressed;
    }
}

File: PreEZ_segmentation_v1.5/addons/SpecificKeyListener.m
classdef SpecificKeyListener < handle
    properties
        keyPressed = false;
    end
    
    methods
        function obj = SpecificKeyListener()
            obj = obj@handle();
        end
        
        function keyTyped(obj, event)
            if event.getKeyChar() == 'x'
                obj.keyPressed = true;
            end
        end
    end
end
